pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -363,21 +365,20 @@ func (s *Summarizer) loadControlsFromFile(filePath string) (*kb.Controls, error)
 	if err != nil {
 		return nil, fmt.Errorf("error reading file %+v: %v", filePath, err)
 	}
-	err = yaml.Unmarshal(fileContents, controls)
-	if err != nil {
+	if err := yaml.Unmarshal(fileContents, controls); err != nil {
 		return nil, fmt.Errorf("error unmarshalling master controls file: %v", err)
 	}
 	logrus.Debugf("filePath: %v, controls: %+v", filePath, controls)
 	return controls, nil
 }
 
-func getNodeTypes() []NodeType {
-	return []NodeType{
-		NodeTypeMaster,
-		NodeTypeEtcd,
-		NodeTypeNode,
-	}
-}
+// func getNodeTypes() []NodeType {
```

Comment:
What's our convention on commented out code?

Reply:
Not sure. The code isn't used and didn't want to remove it. figured I'd comment it out and the primary maintainer could make the call. I'd say for anything rke2 related, it gets removed entirely but not sure for this repo.